server/src/handlers/route.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 CeresDB Project Authors. Licensed under Apache-2.0.
+
+//! route request handler
+
+use std::{collections::HashMap, time::Instant};
+
+use ceresdbproto::storage::RouteRequest;
+use common_util::time::InstantExt;
+use log::info;
+
+use crate::handlers::{error::RouteHandler, prelude::*};
+
+#[derive(Serialize)]
+pub struct RouteResponse {
+    routes: HashMap<String, String>,
+}
+
+pub async fn handle_route<Q: QueryExecutor + 'static>(
+    ctx: &RequestContext,
+    _: InstanceRef<Q>,
+    table: String,
+) -> Result<RouteResponse> {
+    let mut route_map = HashMap::new();
+    if table.is_empty() {
+        return Ok(RouteResponse { routes: route_map });
+    }
+
+    let begin_instant = Instant::now();
+    info!("Route handler try to find route for table:{table}");
```

Comment:
How about removing the logs on L29 and L44?

Reply:
Well, L29 should be removed.  I think L44 can be reserved, and change log level to `debug`, because sometime I want to measure the cost of route.

Reply:
You can use prometheus to collect some metrics.

Reply:
well, I will remove it .